fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/NOOPFilter.java

```diff
@@ -33,12 +31,7 @@
 public class NOOPFilter implements EventFilter {
 
     @Override
-    public Predicate<Event> getFilter(final Session session) {
-        return this;
-    }
-
-    @Override
-    public boolean apply(final Event event) {
+    public boolean test(final Event event) {
```

Comment:
I'll add a commit with the excision. Monkeys? That's all Fedora needs. Monkeys in the codebase.